packages/sdk/src/types/Vouchers.ts

```diff
@@ -0,0 +1,203 @@
+	redemption?: {
+		object: 'list'
+		quantity?: number
+		redeemed_quantity: number
+		data_ref: 'redemption_entries'
+		redemption_entries: $FixMe[]
+		total: number
+		url: string
+	}
+	active: boolean
+	additional_info?: string
+	metadata?: Record<string, any>
+	assets?: {
+		qr?: {
+			id: string
+			url: string
+		}
+		barcode?: {
+			id: string
+			url: string
+		}
+	}
+	is_referral_code: boolean
+	referrer_id?: string
+	holder_id?: string
+	updated_at?: string
+	object: 'voucher'
+	validation_rules_assignments: {
+		object: 'list'
+		total: number
+		data_ref: 'data'
+		data?: {
+			id: string
+			rule_id?: string
+			related_object_id?: string
+			related_object_type?: string
+			created_at: string
+			object: 'validation_rules_assignment'
+		}[]
+	}
+}
+
+export interface VouchersQualificationExamineBody {
+	customer?: Omit<SimpleCustomer, 'object'> & { description: string }
+	order?: Pick<OrdersGetResponse, 'id' | 'source_id' | 'amount' | 'items' | 'metadata'>
+	reward?: {
+		id: string
+		assignment_id?: string
+	}
+	metadata?: Record<string, any>
+}
+
+export interface VouchersQualificationExamineParams {
+	audienceRulesOnly?: boolean
+	order?: 'created_at' | '-created_at' | 'updated_at' | '-updated_at'
+	limit?: number
+}
+
+export interface VouchersQualificationExamineResponse {
+	object: 'list'
+	total: number
+	data_ref: 'data'
+	data?: VouchersResponse[]
+}
+
+export interface VouchersCreateParameters {
+	code?: string
+	code_config?: {
+		length?: number
+		charset?: string
+		pattern?: string
+		prefix?: string
+		suffix?: string
+	}
+	redemption?: {
+		quantity: number
+	}
+}
+
+export type VouchersCreate = VouchersCreateParameters &
+	Pick<
+		VouchersResponse,
+		'type' | 'discount' | 'gift' | 'category' | 'additional_info' | 'start_date' | 'expiration_date' | 'metadata'
+	>
+
+export type VouchersCreateResponse = Omit<VouchersResponse, 'validation_rules_assignments'>
+
+export type VouchersGetResponse = VouchersResponse
+
+export interface VouchersUpdate {
+	code?: string
+	category?: string
+	start_date?: string
+	expiration_date?: string
+	active?: boolean
+	additional_info?: string
+	metadata?: Record<string, any>
+	gift?: {
+		amount: number
+	}
+}
+
+export type VouchersUpdateResponse = VouchersResponse
+
+export interface VouchersDeleteParams {
+	force?: boolean
+}
+
+export interface VouchersListParams {
+	limit?: number
+	page?: number
+	category?: string
+	campaign?: string
+	customer?: string
+	created_at?: {
+		after?: string
+		before?: string
+	}
+	updated_at?: {
+		after?: string
+		before?: string
+	}
+}
+
+export interface VouchersListResponse {
+	object: 'list'
+	total: number
+	data_ref: 'vouchers'
+	vouchers: VouchersResponse[]
+}
+
+export interface VouchersEnableParams {
+	code?: string
+}
+
+export type VouchersEnableResponse = VouchersResponse
+
+export interface VouchersDisableParams {
+	code?: string
+}
+
+export type VouchersDisableResponse = VouchersResponse
+
+export interface VouchersImport {
```

Comment:
It's missing `redemption?` property similar to the one when creating new voucher